src/components/RelatiesKoppelingenVisualisatie/RelatiesKoppelingenVisualisatie.js

```diff
@@ -7,6 +7,19 @@ import generateVerordeningsPosition from './../../utils/generateVerordeningsPosi
 // Import Context
 import GraphContext from './../../App/GraphContext'
 
+/**
+ * Component that renders the RelatiesKoppelingenVisualisatie component, which displays a Netwerkvisualisatie map which shows the beleids objecten connections.
+ *
+ * @component
+ *
+ * @param {object} beleidsObject -  Parameter containing the information of a beleid
+ * @param {Array} connectionProperties - Parameter containing a collection of connection properties.
+ * @param {Array} connectionPropertiesColors - Parameter containing a collection of connection property colors.
```

Comment:
Object type

Reply:
I've changed the parameter description and type to object.